charts/fission-all/values.yaml

```diff
@@ -457,6 +457,7 @@ serviceMonitor:
   ##namespace in which you want to deploy servicemonitor
   ##
   namespace: ""
+  additionalServiceMonitorLabels: {}
```

Comment:
Can we add a service monitor for all these components ?

I see as of now we have covered 4 out of 6.

```
charts/fission-all/templates/storagesvc/deployment.yaml
21:        prometheus.io/scrape: "true"

charts/fission-all/templates/mqt-fission-kafka/deployment.yaml
22:        prometheus.io/scrape: "true"

charts/fission-all/templates/buildermgr/deployment.yaml
18:        prometheus.io/scrape: "true"

charts/fission-all/templates/controller/deployment.yaml
21:        prometheus.io/scrape: "true"

charts/fission-all/templates/executor/deployment.yaml
18:        prometheus.io/scrape: "true"

charts/fission-all/templates/router/deployment.yaml
27:        prometheus.io/scrape: "true"
```

Reply:
Added servicemontiors for the remaining components